### PR TITLE
Add example for sqlpackage extract

### PR DESCRIPTION
**Pull request by sredman**
Add an example usage of sqlpackage extract to create a schema-only .dacpac

### CODE REVIEW

**Comment by PRMerger6** [comment]
@sredman : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/tools/sqlpackage/sqlpackage-extract.md [hunk 1]
```diff
@@ -81,6 +81,13 @@ SqlPackage {parameters}{properties}{SQLCMD Variables}
 |**/p:**| TempDirectoryForTableData=(STRING)|Specifies the temporary directory used to buffer table data before being written to the package file.|
 |**/p:**|VerifyExtraction=(BOOLEAN 'True')|Specifies whether the extracted schema model should be verified.|
 
+## Examples
+
+Create a schema-only .dacpac for database upgrades, etc.
+```cmd
+sqlpackage.exe extract /TargetFile:{filename}.dacpac /DiagnosticsFile:{logFile}.log /p:ExtractAllTableData=false /p:VerifyExtraction=true /SourceServerName:{serverFQDN} /SourceDatabaseName:{databaseName} /SourceUser:{username} /SourcePassword{password}
```
Comment (dzsquared) [inline docs/tools/sqlpackage/sqlpackage-extract.md:88]: /SourcePassword:{password}

**Review by dzsquared — CHANGES_REQUESTED** [review]

File: docs/tools/sqlpackage/sqlpackage-extract.md [hunk 2]
```diff
@@ -81,6 +81,13 @@ SqlPackage {parameters}{properties}{SQLCMD Variables}
 |**/p:**| TempDirectoryForTableData=(STRING)|Specifies the temporary directory used to buffer table data before being written to the package file.|
 |**/p:**|VerifyExtraction=(BOOLEAN 'True')|Specifies whether the extracted schema model should be verified.|
 
+## Examples
+
+Create a schema-only .dacpac for database upgrades, etc.
```
Comment (dzsquared) [inline docs/tools/sqlpackage/sqlpackage-extract.md:86]: + "using SQL authentication"

**Comment by PRMerger18** [comment]
@sredman : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dzsquared** [comment]
#sign-off

**Comment by PRMerger14** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**